pkg/dbmate/db.go

```diff
@@ -330,6 +335,11 @@ func (db *DB) Migrate() error {
 		_ = db.DumpSchema()
 	}
 
+	err = drv.ReleaseChangeLock(sqlDB)
```

Comment:
this should probably live inside a `defer` block

Reply:
How do i do that while still checking for errors though? 😬 panicking seems a little dramatic